README.md

```diff
@@ -17,13 +17,26 @@
 Some Android plugin versions have issues with Gradle's build cache feature. When applied to an Android project this plugin applies workarounds for these issues based on the Android plugin and Gradle versions. For other versions, please see [older versions.](#older-android-gradle-plugin-versions)
 
 * Supported Gradle versions: 7.0+
-* Supported Android Gradle Plugin versions: 7.0.4, 7.1.3, 7.2.2, 7.3.1, 7.4.2, 8.0.2, 8.1.0 and 8.2.0-alpha16
+* Supported Android Gradle Plugin versions: 7.0+
```

Comment:
Did you write in the doc that there is also a max supported version? I added a comment in the doc, but what are the pros and cons of having that max version?
If we do have a max version, we need to document it.

Reply:
after discussion in the doc, we dropped the max supported version and the patch labels on the supported android versions